### ISSUE TITLE
[0.6.0] "Unexpected `}`" error when using comments in interpolation strings

### ISSUE THREAD

**Issue by rgehan** [hudochenkov/postcss-styled-syntax#26]
Hi!

Since v0.6.0, it seems having comments inside interpolation strings causes issues.

Repro test case:
```js
const parse = require('../parse');

test('it should not fail with comments inside interpolation', () => {
	expect(() => {
		parse('let Component = styled.div`color: ${textColor /* very good color */}`;')
	}).not.toThrow();
});
```

```
expect(received).not.toThrow()

Error name:    "CssSyntaxError"
Error message: "<css input>:1:41: Unexpected }"
```

This fails on v0.6.0, but doesn't fail on v0.5.0.

Running a quick `git bisect`, I find 6c9cdaab23bccd980610c852131edffe6db22fae as the culprit commit.

**Comment by rgehan** [comment]
Just in case, I filed a bug with `postcss` too: https://github.com/postcss/postcss/issues/1907

**Comment by hudochenkov** [comment]
Thank you for reporting! Fixed in `0.6.1`.
